bitnami/concourse/templates/_helpers.tpl

```diff
@@ -0,0 +1,135 @@
+{{/*
+Return the proper web image name
+*/}}
+{{- define "concourse.image" -}}
+{{ include "common.images.image" (dict "imageRoot" .Values.concourse.image "global" .Values.global) }}
+{{- end -}}
+
+{{/*
+Return the proper image name (for the init container volume-permissions image)
+*/}}
+{{- define "concourse.volumePermissions.image" -}}
+{{- include "common.images.image" ( dict "imageRoot" .Values.volumePermissions.image "global" .Values.global ) -}}
+{{- end -}}
+
+{{/*
+Return the proper Docker Image Registry Secret Names
+*/}}
+{{- define "concourse.imagePullSecrets" -}}
+{{- include "common.images.pullSecrets" (dict "images" (list .Values.concourse.image .Values.volumePermissions.image) "global" .Values.global) -}}
+{{- end -}}
+
+{{/*
+Create the name of the service account to use
+*/}}
+{{- define "concourse.serviceAccountName" -}}
```

Comment:
This is not used in the yaml. You should create another one for the worker and edit the rbac objects